fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+    
+      id:
+        type: integer
+        format: int64
+        description: Unique numeric identifier for the metafield.
+        
+      key:
+        type: string
+        description: Identifier for the metafield (maximum of 30 characters).
+      namespace:
+        type: string
+        description: Container for a set of metadata. Namespaces help distinguish between metadata you created against metadata created by another individual with a similar namespace (maximum of 20 characters).
+        
+      owner_id:
+        type: integer
+        format: int64
+        description: A unique numeric identifier for the metafield's owner.
+      owner_resource:
+        type: string
+        description: Unique id for that particular resource.
+        
+      value: 
+        type: integer
+        format: int64
+        description: Information to be stored as metadata.
+        
+      value_type:
+        type: string
+        description: States whether the information in the value is stored as a 'string' or 'integer.'
+        enum:
+        - string
+        - integer
+      updated_at: 
+        type: string
+        format: date-time
+        description: The date and time when the metafield was published. The API returns this value in ISO 8601 format.
+        
+
+  tax_line:
+    type: object
+
+    properties:
+      title:
+        type: string
+        description: The name of the tax.
+        readOnly: true
+      rate:
+        type: string
+        description: The rate of tax to be applied.
+        readOnly: true
+      price:
+        type: string
+        description: The amount of tax to be charged.
+        readOnly: true
+
+  applied_discount:
+    type: object
+    description: Discount which will be applied to the line item or the overall order. A draft order line item can have one discount. A draft order can have one order-level discount. 
+
+    required:
+    - title
+    - amount
+    - value
+    - value_type
+
+    properties:
+      
+      title:
+        type: string
+        description: Title of the discount.
+        readOnly: false
+      
+      description:
+        type: string
+        description: Reason for the discount.
+        readOnly: false
+      value:
+        type: string
+        description: The value of the discount. If the type of the discount is `fixed_amount`, then this is a fixed dollar amount. If the type is `percentage`, then this is the percentage.
+        readOnly: false
+      value_type:
+        type: string
+        description: The type of the discount.
+        enum:
+        - percentage
+        - fixed_amount 
+        
+        readOnly: false
+
+      amount:
+        type: string
+        description: The applied amount of the discount.
+        readOnly: false
+
+  customer:
+    type: object
+    description: Customer associated with the draft order.
+    readOnly: true
+    properties:
+        
+        
```

Comment:
missing default address which I think might be really important for how address loading on drafts works. Not blocking alpha email can be done later this week.